### ISSUE TITLE
RFE: Google translate

### ISSUE THREAD

**Issue by ukdtom** [pannal/Sub-Zero.bundle#282]
Since SZ is looking over the shoulders of SubtitleEdit, it would be cool, that if a sub can't be found for a req. language, then download english sub, and do a google translate.

Later, if a sub is released for req. language, download and replace like normal

**Comment by pannal** [comment]
Yeah that'd be nice. But API access to Google Translate costs money nowadays. There are python libraries out there, that circumvent that restriction, but they may be blocked any time.

**Comment by leoheck** [comment]
I'm almost sure there must be some alternative for Google translator.

**Comment by ukdtom** [comment]
Would help, if you provided a list of alternatives ;)

**Comment by leoheck** [comment]
:) @ukdtom you are absolutely right but I don't know much how things are implemented here. So I can't add much in this way but someone else can! I was just adding that it could be done without google stuff, for example.

**Comment by ukdtom** [comment]
It's not a matter of how it's implemented, since it's currently not a part of SZ.

So if you know of a provider, that can translate a sentence one by one into all languages, then please speak up
